_extensions/bluesky-comments/bluesky-resolve.lua

```diff
@@ -0,0 +1,107 @@
+local BlueskyAPI = {}
```

Comment:
Maybe we should call this file `bluesky-api` instead of `resolve` if we're storing into and returning the `BlueSkyAPI` table.

Reply:
yeah great point. It only does resolving right now but naming the module was awkward. Let's just call the whole the `bluesky-api`.

Reply:
011b2ab2fb5c8653cf4198bda5f7281a69464be3